sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -220,8 +215,7 @@ def get_trace_parent(self):
         return self.to_header()["traceparent"]
 
     @classmethod
-    def link(cls, traceparent, attributes=None):
-        # type: (str, Attributes) -> None
+    def link(cls, traceparent: str, attributes: Attributes = None) -> None:
```

Comment:
```suggestion
    def link(cls, traceparent: str, attributes: Optional[Attributes] = None) -> None:
```

Reply:
Typing for `attributes` follows how the `opentelemetry` package defines it where the `Optional` is part of `Attributes` (https://github.com/Azure/azure-sdk-for-python/blob/e786a17cd251327f9a5962ca825806aff9200f49/sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py#L31)

Reply:
ah I missed that